frontend/src/hocs/withScreencast/components/VideoRecordedDialog/style.scss

```diff
@@ -16,62 +16,58 @@
 
 @import "src/assets/css/theme";
 
-.streamlit-dialog.screencast {
```

Comment:
I'm not sure this is the right move? While this is imported specifically in the VideoRecordedDialog component, this still gets compiled into a css file for the page. It does not look like we do [the fancy webpack term] with dynamic loading. With this change, these styles would then get applied to all other modal bodies. While most of them either make sense globally or is random enough, `.download-button` could be problematic in the future. If we do prefer this to ax the `.streamlit-dialog.screencast` parent, I would vote to move this into our assets folder instead for future developers to not go wondering why does my `download-button` have a `margin-top` though whole other topic you'll be addressing :)

Reply:
oh. you're probably right. Let me relook into this.